sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -614,3 +614,16 @@ CREATE TABLE `fee_schedule` (
     UNIQUE KEY `ins_plan_code_mod_type_date` (`insurance_company_id`, `plan`, `code`, `modifier`, `type`, `effective_date`)
 ) ENGINE=InnoDb AUTO_INCREMENT=1;
 #EndIf
+
```

Comment:
hi @juggernautsei, no need for this file

Reply:
That is not a part of my PR.

Reply:
drop the changes in this script

Reply:
to remove this from the PR just copy the [raw file](https://raw.githubusercontent.com/openemr/openemr/master/sql/7_0_1-to-7_0_2_upgrade.sql) into the file in this branch

Reply:
This still needs to happen.

Reply:
@adunsulag 
@stephenwaite removed this. It was not apart of my PR.

Reply:
Still here. You can't simply remove, you have to commit the correct version.
You really need to pull and merge master in your local then push.

You know, better yet @juggernautsei, close this and push new PR with correct files for the feature option. I'm not sure what happened to this PR but it is a mess. For sure get rid of new table and list options so really no need to push any database/upgrade changes.
Since now in globals with hard coded select options I foresee only two or three files for feature.
This IMO is easiest way to get this in....

Reply:
@juggernautsei Did you see this above?